app-config.yaml

```diff
@@ -69,14 +69,16 @@ organization:
   name: My Company
 
 # Reference documentation http://backstage.io/docs/features/techdocs/configuration
+# publisher:
+# type: 'local' # Alternatives - 'googleGcs' or 'awsS3' or 'azureStorage'. Read documentation for using alternatives.
```

Comment:
this line seems to just duplicate what is below at line 81? was this intentional? if not, suggest removing these two.

Reply:
It was not intentional, I will remove